### ISSUE TITLE
Additional option for Growl notifications

### ISSUE THREAD

**Issue by ghost** [bdkjones/CodeKit-1-Old#294]
It would be nice to have a Sub-CheckBox for an additional Growl option in the _Preferences | General -> Other_ :

[  ] Bring CodeKit to …
[X] Show Growl notifications
--- **[X] Only for errors / Show only for errors / Errors only**
…

**Comment by TuckerWhitehouse** [comment]
+1

**Comment by zanona** [comment]
+1

**Comment by alampros** [comment]
Can't you just disable the non-error growls from the growl app?

**Comment by zanona** [comment]
@alampros I'm afraid codekit offers growl support without the need of having growl app installed. I don't have it installed myself. so I guess it wouldn't be possible to achieve such solution, but perhaps I'm wrong.

**Comment by ghost** [comment]
@alampros You're right. This is possible. Thank you for the advice. IMHO in the long run this should only be a fallback solution. Other apps like Twitter allow you to change notification settings in the own preferences.
